### ISSUE TITLE
Feature: Support goto the specified line when open file, just like vim

### ISSUE THREAD

**Issue by amitbha** [charmbracelet/glow#238]
Just like ` vim +[num] file ` 
It's useful when searching .md's content with **grep**, then view the detail with the line number.

**Comment by tusktenon** [comment]
I agree! This would also be great for the [glow.nvim](https://github.com/ellisonleao/glow.nvim) Neovim plugin: you could open the Glow preview to the line your cursor is on.
